all/entrypoint.sh

```diff
@@ -136,6 +136,7 @@ esac
 
 PROTO_INCLUDE="-I /usr/include/ \
     -I /usr/local/include/ \
+    -I /usr/include/grpc-gateway/ \
```

Comment:
you should be able to just make this `-I/go/src/github.com/grpc-ecosystem/grpc-gateway`